docs/interpreter/jdbc.md

```diff
@@ -156,6 +156,25 @@ The last step is **Dependency Setting**. Since Zeppelin only includes `PostgreSQ
+| timeBetweenEvictionRunsMillis                                                                                                          | -1L           | The number of milliseconds to sleep between runs of the idle object evictor thread. When non-positive, no idle object evictor thread will be run.                                                                                                                                     |
+| maxWaitMillis                                                                                                                          | -lL           | The maximum number of milliseconds that the pool will wait (when there are no available connections) for a connection to be returned before throwing an exception, or -1 to wait indefinitely.                                                                                        |
+| maxIdle                                                                                                                                | 8             | The maximum number of connections that can remain idle in the pool, without extra ones being released, or negative for no limit.                                                                                                                                                      |
+| minIdle                                                                                                                                | 0             | The minimum number of connections that can remain idle in the pool, without extra ones being created, or zero to create none.                                                                                                                                                         |
+| maxTotal                                                                                                                               | -1            | The maximum number of active connections that can be allocated from this pool at the same time, or negative for no limit.                                                                                                                                                             |
+| validationQuery                                                                                                                        | show database | The SQL query that will be used to validate connections from this pool before returning them to the caller. If specified, this query MUST be an SQL SELECT statement that returns at least one row. If not specified, connections will be validation by calling the isValid() method. |
+
+
```

Comment:
You can use html table to render this. Here's one example of spark interpreter.

https://github.com/apache/zeppelin/blob/master/docs/interpreter/spark.md

Reply:
@zjffdu Okay, I have updated it with the html table syntax

Reply:
> You can use html table to render this. Here's one example of spark interpreter.
> 
> https://github.com/apache/zeppelin/blob/master/docs/interpreter/spark.md

@zjffdu OK, I have updated the datasource pool docs in the  jdbc.md with the html table syntax.